src/components/ToggleSwitch/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+// Constants
+import {ToggleSwitchLabelPositioningValues} from '@Constants';
+
+type ToggleSwitchProps = {
+    active: boolean;
+    onChange: (active: boolean) => void;
+    label?: string;
+    labelPositioning?: ToggleSwitchLabelPositioning;
```

Comment:
No, otherwise when using this component, we'll always have to create the enum with the same structure as the library which isn't good. The reusability shouldn't depend on internal details, I think.